features/step_definitions/steps.rb

```diff
@@ -32,5 +32,9 @@ def path(pagename)
       new_user_registration_path
     when 'create menu'
       new_menu_path
+    when 'restaurant edit details'
+      edit_restaurant_path(@restaurant)
+    else
+      you_have_not_defined_a_path
```

Comment:
Yeah, the format was on purpose, not to make it look for a method though.  But I will replace it with a string.